src/process.cr

```diff
@@ -294,6 +294,14 @@ class Process
     when IO::FileDescriptor
       stdio
     when IO
+      if stdio.closed?
+        if dst_io == STDIN
+          return File.open(File::NULL, "r").tap(&.close)
+        else
+          return File.open(File::NULL, "w").tap(&.close)
+        end
```

Comment:
maybe:

```crystal
mode = dst_io == STDIN ? "r" : "w"
return File.open(File::NULL, mode).tap(&.close)
```

Reply:
This is copied directly from a different location in this file. I'd like to refactor them together in a follow-up.